server/app/views/admin/questions/QuestionConfig.java

```diff
@@ -235,6 +235,8 @@ private QuestionConfig addMultiOptionQuestionFields(
                   LocalizedQuestionOption.create(
                       multiOptionQuestionForm.getOptionIds().get(i),
                       optionIndex,
+                      // TODO(#4862): Use the admin name from the form here
```

Comment:
Here and below, we just pass the provided option text into`LocalizedQuestionOption.create()` as the admin name for now. This is what we already do in other calls to create(), while we migrate all callsites to also use adminNames.